recipes/adam-robotics/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "adam-robotics" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ami-iit/ADAM/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4a726ac0c0ce6735eb51be30f512583d63721a270d16cb74738f78615540a6cb
+  patches:
+    - 40.patch
```

Comment:
Can you add a comment here explaining the patch and/or if it was submitted upstream?

Reply:
Yes, done at https://github.com/conda-forge/staged-recipes/pull/21894/commits/ed5d32a332a63a88f1b19fbd8f055385eba26241 , thanks!